cves/kernel/CVE-2019-9503.yml

```diff
@@ -377,8 +409,10 @@ order_of_operations:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: false
+  note: |
+    The fix for this vulnerability was adding a forgotten check, so there was no
```

Comment:
Again, think of the addition of a new check does change the order of operations

Reply:
Although I see your point, the ORDER of the operations wasn't a problem. It was the operations themselves. So although the number of operations has changed, the original code is still intact, there is simply an additional operation now. So I don't think that constitutes a change in order of operations.